metricbeat/module/kibana/stats/stats_integration_test.go

```diff
@@ -28,15 +28,14 @@ import (
 	"github.com/stretchr/testify/assert"
 
 	"github.com/elastic/beats/libbeat/common"
-	"github.com/elastic/beats/libbeat/tests/compose"
 
 	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
 	"github.com/elastic/beats/metricbeat/module/kibana"
 	"github.com/elastic/beats/metricbeat/module/kibana/mtest"
 )
 
 func TestFetch(t *testing.T) {
-	compose.EnsureUpWithTimeout(t, 600, "elasticsearch", "kibana")
+	// compose.EnsureUpWithTimeout(t, 600, "elasticsearch", "kibana")
```

Comment:
Leftover? To skip these during testing without uncommenting you can use `NO_COMPOSE=1`

Reply:
OOOOOPS! Removing... 😄